app/services/ldap_group_cleaner.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class LdapGroupCleaner
+  class Error < RuntimeError; end
+
+  class << self
+    def call(ldap_record)
+      call!(ldap_record)
+    rescue Error
+      nil
```

Comment:
@rschenk we ought to log these errors. If ldap is returning oddly named groups, we'd want to know about it.

Reply:
Where should we log them? Do we have a tool like bugsnag or similar? Seems like just writing them to the Rails log, they'll probably never be seen again

Reply:
We use splunk for logging, we could do forensics there if we need to.

Reply:
@rschenk Rails log at the `error` is fine for now, but yes, we need a way of being alerted if these things are showing up. I'm imagining we can grep for LdapGroupCleaner::Error via Splunk.